albumentations/augmentations/geometric/transforms.py

```diff
@@ -93,8 +100,8 @@ def get_params(self):
             "dy": random.uniform(self.shift_limit_y[0], self.shift_limit_y[1]),
         }
 
-    def apply_to_bbox(self, bbox, angle, scale, dx, dy, **params):
-        return F.bbox_shift_scale_rotate(bbox, angle, scale, dx, dy, **params)
+    def apply_to_bbox(self, bbox, angle, scale, dx, dy, rotate_method="largest_box", **params):
+        return F.bbox_shift_scale_rotate(bbox, angle, scale, dx, dy, rotate_method, **params)
```

Comment:
Don't add a `method` as an argument, just specify it using `method=self.method`.